src/main/resources/application.properties

```diff
@@ -1 +1,2 @@
 # spring.config.import=optional:file:.env[.properties]
+jwt.secret=1abcdascabADGAGEbcdabc531dab1235cd!@)*$(abcd
```

Comment:
Hmm... secret out lol

Reply:
Yep lmao
Shall replace it for actual deployment